pkg/controller/mcmhub/hub.go

```diff
@@ -75,7 +71,7 @@ func (r *ReconcileSubscription) doMCMHubReconcile(sub *appv1alpha1.Subscription)
 
 	chnAnnotations := primaryChannel.GetAnnotations()
 
-	if chnAnnotations[appv1.AnnotationResourceReconcileLevel] != "" {
+	if chnAnnotations[appv1alpha1.AnnotationResourceReconcileLevel] != "" {
```

Comment:
We should keep `appv1 "open-cluster-management.io/multicloud-operators-subscription/pkg/apis/apps/v1"` and delete the `appv1alpha1 "open-cluster-management.io/multicloud-operators-subscription/pkg/apis/apps/v1"`

Because appv1 is the current version. The alpha is the old version that we no longer use.

Reply:
Thank you will change this